client/components/data/query-verticals/test/index.js

```diff
@@ -34,20 +34,18 @@ describe( 'QueryVerticals', () => {
 		expect( requestVerticals ).not.toHaveBeenCalled();
 	} );
 
-	test( 'should call request on update if isFetched is false.', () => {
+	test( 'should not call request on mount if search term is falsey.', () => {
 		const requestVerticals = jest.fn();
-
 		const wrapped = shallow( <QueryVerticals requestVerticals={ requestVerticals } /> );
-
 		const updatedProps = {
-			searchTerm: 'Foo',
+			searchTerm: undefined,
 			limit: 7,
-			isFetched: false,
+			isFetched: true,
```

Comment:
Since `isFetched` can also prevent a request from happening, this should be `false` to clearly indicate that `requestVerticals` is not been called because of `searchTerm`

Reply:
Ugh, thanks for catching this. I think this was a mistake that I accidentally checked it :( 

Will revert.